docs/faq/README.md

```diff
@@ -87,6 +64,42 @@ Regardless of storage option used, IPAM controller expects a directory volume mo
 
 In the mount directory, rename or remove a file named `cis_ipam.sqlite3`.
 
+## Troubleshooting
+
+### How to troubleshoot FIC pod logs ?
+
+`kubectl logs deploy/<name-of-ipam-deployment> -n kube-system -f`
+
+### Error - `Unable to Update IPAM: kube-system/***  Error: ipams.fic.f5.com "***" not found`
+
+* Verify cluster role permissions for ipam resource.
+
+  - `kubectl describe clusterrole <name> -n kube-system | grep ipam`
+
+* Delete existing IPAM CRD and re-start both CIS and IPAM pods
+  * Scale down CIS controller Deployment
+    * `kubectl scale deploy/bigip-controller -n kube-system --replicas=0`
+  * Scale down FIC controller Deployment
+    * `kubectl scale deploy/ipam-controller -n kube-system --replicas=0`
+  * `kubectl delete crd ipams.fic.f5.com`
```

Comment:
Let's list the CRD and then delete them. This will make clear which crd we are deleting. Hide other CRDs with ". . . " and show only that CRD that we want to delete.